vagrant/README.md

```diff
@@ -1,3 +1,36 @@
-Vagrant VM to build and run Heron
-=================================
-vagrant up
\ No newline at end of file
+# Vagrant image for running Heron examples
+
+## Overview
+
+With the help of this image one should be able to test their topologies on Heron.
+
+## Prerequisites
+
+There are two requirements for a host machine. Please refer to official docs for installation instructions:
+
+- [Vagrant](http://vagrantup.com) - required for running cluster VMs
+
+## Contents
+
+- 1x Mesos Master node
+- Configurable number of Mesos Slaves/Agents (via `SLAVES` parameter in `Vagrantfile`)
+- Zookeeper running at `master:2181`
+- Marathon running at `master:8080`
+- Aurora scheduler (plain Mesos cluster supported by changing `SCHEDULER` parameter in `Vagrantfile` to `mesos`) running on master
```

Comment:
I don't think `scheduler` parameter still works here. As basically we don't install anything to run a Mesos scheduler. I think would be nicer to have `Aurora` parameter instead, set it to true by default so that the users would be able to disable this if needed.